pkg/releaselock/controller.go

```diff
@@ -224,7 +241,7 @@ func (c *LockReleaseController) verifyNodeExists(node *corev1.Node, expectedGCEI
 	return false, nil
 }
 
-func (c *LockReleaseController) listNodes(ctx context.Context) (map[string]*corev1.Node, error) {
+func (c *LockReleaseController) ListNodes(ctx context.Context) (map[string]*corev1.Node, error) {
```

Comment:
I think this function can be removed since we no longer list nodes in the lock release controller

Reply:
done in next version